### PR TITLE
Newhorizons data

### PR DESCRIPTION
**Pull request by paarongiroux**
## Description

Moved newhorizons translations files from the data area into the source code. Updated all references to these files.
## Related Issue

#3727 
## Motivation and Context

part of moving isis test data
## How Has This Been Tested?

All newhorizons app tests are passing
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read and agree to abide by the [Code of Conduct](https://usgs-astrogeology.github.io/code/)
- [ ] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [x] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

File: isis/src/newhorizons/apps/leisa2isis/NewHorizonsLeisaArchive_fit.trn [hunk 1]
```diff
@@ -0,0 +1,42 @@
+#
```
Comment (paarongiroux) [inline isis/src/newhorizons/apps/leisa2isis/NewHorizonsLeisaArchive_fit.trn:1]: A lot of these files had _fit at the end of the filename. Wasn't sure what the significance of that is (if any), or whether or not I should have left it in. Feedback is appreciated

Comment (krlberry) [reply isis/src/newhorizons/apps/leisa2isis/NewHorizonsLeisaArchive_fit.trn:1]: My best guess is that it stands for "FITS" as in the file type of the input. There's no real reason to leave them on, but I also think it's okay that you didn't take them off.

File: isis/src/newhorizons/apps/leisa2isis/NewHorizonsLeisaKernels_fit.trn [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# First attempt at LEISA Kernel translation
```
Comment (krlberry) [inline isis/src/newhorizons/apps/leisa2isis/NewHorizonsLeisaKernels_fit.trn]: Any chance you could remove this comment while you're in here?

**Review by krlberry — APPROVED** [review]
:+1: This all looks good. If you can, could you take out that one comment I noticed?

**Pull request merged**